talknet/demoTalkNet.py

```
@@ -1,726 +1,726 @@
+	os.makedirs(pyaviPath, exist_ok = True) # The path for the input video, input audio, output video
+	os.makedirs(pyframesPath, exist_ok = True) # Save all the video frames
+	os.makedirs(pyworkPath, exist_ok = True) # Save the results in this process by the pckl method
+	os.makedirs(pycropPath, exist_ok = True) # Save the detected face clips (audio+video) in this process
+	
+	STORE_FRAMES_IN_MEMORY_THRESHOLD = in_memory_threshold
+
+    # Extract video information
+    # get video duration
+	print("Extracting video information...")
+	command = ("ffprobe -v error -select_streams v:0 -count_packets -show_entries stream=nb_read_packets -of csv=p=0 '%s'" % video_path)
+	video_num_frames = float(subprocess.check_output(command, shell=True))
```

Ruff (0.13.3)

`subprocess` call with `shell=True` identified, security issue

(S602)